internal/ingress/controller/controller.go

```diff
@@ -580,7 +580,7 @@ func (n *NGINXController) getBackendServers(ingresses []*ingress.Ingress) ([]*in
 		for _, server := range servers {
 			for _, location := range server.Locations {
 				if upstream.Name == location.Backend {
-					if len(upstream.Endpoints) == 0 {
+					if len(upstream.Endpoints) == 0 || len(location.CustomHTTPErrors) != 0 {
 						klog.V(3).Infof("Upstream %q has no active Endpoint", upstream.Name)
```

Comment:
I think this need to be replaced - it is not going to be true when `len(upstream.Endpoints) != 0 and len(location.CustomHTTPErrors) != 0`

Reply:
it should be true in that case, right? maybe I'm not following, but if you mean `false || true` it should eval to true